### PR TITLE
[DEV-7071] Fixed dearth of FKs in treasury_appropriation_account

### PR DESCRIPTION
**Pull request by tony-sappe**
**Description:**
Many many issues appeared from #3014 after switching from "Funding Agency" to "Awarding Agency" for About the Data [DEV-6958](https://federal-spending-transparency.atlassian.net/browse/DEV-6958).

**Technical details:**
Updated the TAS ETL so `awarding_toptier_agency_id` wasn't null (went from over 90% null to 0% null). Fix was for `update_agencies.py`

**Requirements for PR merge:**

1. [x] Unit & integration tests updated
2. [x] API documentation updated (N/A)
3. [x] Necessary PR reviewers:
    - [x] Backend
4. [x] Matview impact assessment completed (N/A)
5. [x] Frontend impact assessment completed (N/A)
6. [x] Data validation completed
7. [x] Appropriate Operations ticket(s) created
8. [x] Jira Ticket [DEV-7071](https://federal-spending-transparency.atlassian.net/browse/DEV-7071):
    - [x] Link to this Pull-Request
    - [x] Performance evaluation of affected Script (no observed change)
    - [x] Before / After data comparison

**Area for explaining above N/A when needed:**
```
No changes to API schema nor materialized views
```

### CODE REVIEW

File: usaspending_api/reporting/management/commands/populate_reporting_agency_overview.py [hunk 1]
```diff
@@ -223,6 +223,10 @@ class TempTableName(Enum):
                 SUM(obligations_incurred_total_cpe) AS total_dollars_obligated_gtas
             FROM
                 gtas_sf133_balances AS gtas
+             INNER JOIN dabs_submission_window_schedule dabs ON
```
Comment (collinwr) [inline usaspending_api/reporting/management/commands/populate_reporting_agency_overview.py:226]: If you end up needing to do another commit for something else, can you fix the tab alignment here?

Comment (tony-sappe) [reply usaspending_api/reporting/management/commands/populate_reporting_agency_overview.py:226]: Did you find anything else I should address? I don't have plans to push another commit

Comment (collinwr) [reply usaspending_api/reporting/management/commands/populate_reporting_agency_overview.py:226]: Nope, just making a note in case anything else came out of our discussion

**Review by collinwr — APPROVED** [review]
